dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/AssuranceDataPoint.kt

```diff
@@ -1,18 +1,19 @@
 package org.dataland.datalandbackend.model.eutaxonomy
 
 import com.fasterxml.jackson.annotation.JsonProperty
-import org.dataland.datalandbackend.model.CompanyReportReference
+import org.dataland.datalandbackend.interfaces.datapoints.BaseDataPoint
+import org.dataland.datalandbackend.model.documents.ExtendedDocumentReference
 import org.dataland.datalandbackend.model.enums.eutaxonomy.AssuranceOptions
 
 /**
  * --- API model ---
  * Level of assurance for the reported data and information about the assurance provider
  */
-data class AssuranceData(
+data class AssuranceDataPoint(
     @field:JsonProperty(required = true)
-    val assurance: AssuranceOptions,
+    override val value: AssuranceOptions?,
```

Comment:
If the field is required it should not be nullable

Reply:
Agreed and changed